lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -43,22 +44,35 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy {
     allowOnlyPredefinedValues = true;
 
     @Output()
-    optionsChanged: EventEmitter<string[]> = new EventEmitter();
+    optionsChanged: EventEmitter<AutocompleteOption[]> = new EventEmitter();
+
+    @Output()
+    inputChanged: EventEmitter<string> = new EventEmitter();
```

Comment:
Why do we need this emitter? If parent object provides the autocompleteOptions it should be aware of its changes